integration/openshift/lib.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Contain helper functions and variables for the OpenShift integration
+# test scripts.
+#
+
+# ATTENTION: the caller of this lib should have the tests_dir exported
+#            correctly
+if [ ! -d "${tests_dir}" ]; then
+	echo "Unable to find the Kata Container tests directory."
```

Comment:
Nit: `echo >&2 "ERROR: Unable ...`.

Reply:
Also should use Kata ContainerS in plural.